js2r-conveniences.el

```diff
@@ -30,9 +30,9 @@
 
 (require 'js2r-helpers)
 
-(defun js2r-log-this ()
-  "Log of the node at point, adding a 'console.log()' statement."
-  (interactive)
+(defun js2r-log-this (arg)
+  "Log of the node at point, adding a 'console.log()' statement. With a prefix argument ARG, use JSON pretty-printing for logging."
```

Comment:
Use 2 spaces between sentences (well, in this case, a line break would be better).

If you're not using flycheck, maybe you could enable it? It would warn you about such things :)